CrowdStrike/crowdstrike_falcon/ingest/parser.yml

```
@@ -0,0 +1,138 @@
+name: 'crowdstrike falcon edr'
+pipeline:
+  - name: stage1
+    description: ''
+    filter: ''
+    external:
+      name: json.parse-json
+      properties:
+        input_field: '{{original.message}}'
+        output_field: message
+  - name: read_event_date
+    description: ''
+    filter: ''
+    external:
+      name: date.parse
+      properties:
+        input_field: '{{stage1.message.metadata.eventCreationTime/1000}}'
+        output_field: test
+        format: timestamp
+        timezone: UTC
+  - name: set_event_date
+    description: ''
+    filter: ''
+    external: null
+  - name: streamstarted
+    description: ''
+    filter: '{{stage1.message.event.OperationName == "streamStarted"}}'
+    external: null
+  - name: streamstopped
+    description: ''
+    filter: '{{stage1.message.event.OperationName == "streamStopped"}}'
+    external: null
+  - name: detectionsummaryevent
+    description: ''
+    filter: '{{stage1.message.metadata.eventType == "DetectionSummaryEvent"}}'
+    external: null
+  - name: useractivityauditevent
+    description: ''
+    filter: '{{stage1.message.metadata.eventType == "UserActivityAuditEvent"}}'
+    external: null
+  - name: authactivityauditevent
+    description: ''
+    filter: '{{stage1.message.metadata.eventType == "AuthActivityAuditEvent"}}'
+    external: null
+  - name: incidentsummaryevent
+    description: ''
+    filter: '{{stage1.message.metadata.eventType == "IncidentSummaryEvent"}}'
+    external: null
+stages:
+  set_event_date:
+    actions:
+      - set:
+          '@timestamp': '{{read_event_date.test}}'
+          crowdstrike.event_type: '{{stage1.message.metadata.eventType}}'
+        filter: ''
+        name: set
+  streamstarted:
+    actions:
+      - set:
+          source.ip: '{{stage1.message.event.UserIp}}'
+          event.kind: event
+          event.type: start
```

`event.type` should be an array according the specification
```suggestion
          event.type: ["start"]
```